src/cpu/acl/acl_post_ops.hpp

```diff
@@ -138,11 +130,9 @@ struct acl_post_ops_t {
             arm_compute::ActivationLayerInfo &act_info_to_fuse) {
 
         CHECK(base_post_ops.set_default_formats(&dst_md));
-        dst_data_type = dst_md.data_type;
-        // If the first entry is eltwise, we fuse it, except when the datatype
-        // is fp16 because in this case we want to execute the eltwise in fp32.
-        if (base_post_ops.len() >= 1 && base_post_ops.entry_[0].is_eltwise()
-                && dst_data_type != data_type::f16) {
```

Comment:
Do we really need to revert the whole PR? Maybe just remove `dst_data_type != data_type::f16` check here will be enough?

Reply:
Do we need to keep f16->f32 cast logic in `acl_post_ops.cpp` and desc datatype swap in `acl_post_ops.h` implemented in the reverted PR?

Reply:
fixed